integrationtest/initial_release/main.d

```diff
@@ -93,6 +93,7 @@ void runNeptune ( RestAPI gh_test )
     tmp.cmd("mkdir -p " ~ GitPath);
     git.cmd("git init");
     git.cmd("git config neptune.upstream " ~ GitRepo);
+    git.cmd("git config neptune.upstream-remote origin");
```

Comment:
What about using `upstreamremote` like [git-hub](https://github.com/sociomantic-tsunami/git-hub/blob/v1.x.x/man.rst#configuration)?

Reply:
I think it looks terrible :P but I can change it for consistency

Reply:
Thanks for making your command uglier :)